backend/de.metas.business/src/main/java/de/metas/quantity/StockQtyAndUOMQty.java

```diff
@@ -62,7 +62,7 @@ public Quantity getUOMQtyNotNull()
 	}
 
 	@JsonProperty("uomQty")
-	private Quantity getUOMQty()
+	public Quantity getUOMQty()
```

Comment:
Why you need this instead of `getUOMQtyNotNull()` ?

Reply:
correction done